server.ts

```diff
@@ -60,6 +61,7 @@ app.get("/", (req: Request, res: Response, err: any) => {
 app.use("/api", userRoutes);
 app.use("/api", productRoutes);
 app.use("/api", categoryRoutes);
+app.use("/api/order", orderRoutes);
```

Comment:
It is  a good idea to include the main resource name here, will update it for all other endpoints. However, it should be "orders" not "order